### PR TITLE
Update GA4 pageview schema and attributes

### PR DESCRIPTION
**Pull request by AshGDS**
Adds all of the new pageview attributes to our docs.

### CODE REVIEW

File: data/analytics/attributes.yml [hunk 1]
```diff
@@ -1,3 +1,11 @@
+- name: ab_test
+  description: Used in the page_view object to record wether an ab test is running on the page.
```
Comment (andysellick) [inline data/analytics/attributes.yml:2]: Typo: `wether`

File: data/analytics/attributes.yml [hunk 2]
```diff
@@ -22,6 +30,22 @@
...
+  value: true/undefined depending on the presence of data-ga4-cookie-banner on the page
+  example: "true"
+  required: no
+  type: string (boolean)
+  redact: false
+
+- name: devolved_nations_banner
+  description: Used in the page_view object to record the countries listed in a devolved nations banner on the page
+  value: The value of data-ga4-devolved-nations-banner on the page
+  example: "England, Scotland, Wales"
```
Comment (andysellick) [inline data/analytics/attributes.yml:44]: Just to check, does the actual data for this example contain spaces between the countries?

File: data/analytics/attributes.yml [hunk 3]
```diff
@@ -120,6 +152,14 @@
   type: string (number)
   redact: false
 
+- name: intervention
+  description: Used in the page_view object to record the presence of an intervention banner on the page
+  value: true/undefined depending on the presence of data-ga4-intervnetion-banner on the page
```
Comment (andysellick) [inline data/analytics/attributes.yml:157]: Typo: `intervnetion`

File: data/analytics/attributes.yml [hunk 4]
```diff
@@ -185,11 +225,35 @@
   type: string
   redact: false
 
+- name: navigation_list_type
+  description: Used in the page_view object to record the navigation list type
```
Comment (andysellick) [inline data/analytics/attributes.yml:229]: Can we explain any more? This is in the browse, right? Could we something like `Used in the page view object to record the type of page being shown in browse, either 'curated' or 'alphabetical'`? (I think the alternative is alphabetical, but check).

File: data/analytics/attributes.yml [hunk 5]
```diff
@@ -185,11 +225,35 @@
...
+  value: content attribute of meta tag govuk:navigation_list_type
+  example: curated
+  required: no
+  type: string
+  redact: false
+
+- name: navigation_page_type
+  description: Used in the page_view object to record the navigation page type
```
Comment (andysellick) [inline data/analytics/attributes.yml:237]: Can we say any more about this? I'm not certain what the navigation page type is.

**Review by andysellick — COMMENTED** [review]
Nice updating 👍 

Just a few typos and minor comments.

File: data/analytics/attributes.yml (hunk 2 above)
Comment (AshGDS) [reply data/analytics/attributes.yml:44]: @andysellick Yep 👍

File: data/analytics/attributes.yml (hunk 4 above)
Comment (AshGDS) [reply data/analytics/attributes.yml:229]: @andysellick Will update it but I'll have to take your word for it, I can't find anywhere where it gives an example of the values, I can only find the pages from the trello card which have `curated` as the value!

**Comment by AshGDS** [comment]
Thanks @andysellick - should be ready for review again :+1:

File: data/analytics/attributes.yml (hunk 5 above)
Comment (andysellick) [reply data/analytics/attributes.yml:237]: Maybe ask a PA?

Comment (AshGDS) [reply data/analytics/attributes.yml]: @andysellick Have done this now and updated the description as a result, so should be ready to review again :+1:

Comment (andysellick) [reply data/analytics/attributes.yml]: Nice work 👍

**Review by andysellick — APPROVED** [review]
Great stuff, good to see the record getting some improvements 👍

**Pull request merged**